packaging/macos/build_environment

```diff
@@ -1,2 +1,2 @@
-2.3-55d9a17-sdk10.15-macosminimum10.12-x86_64
-86c56d3aba24ec23923ad906f5a13084b0f784c955812448ecb43821fd907d50
+2.3-j00017-a52ffcc9-osx10.11-x86_64-release
```

Comment:
Is `osx10.11` correct? I always struggle with these incomprehensible build environment names.

Reply:
No, but the URL is what it is.